### PR TITLE
Update readme with AutoGluon

### PR DESCRIPTION
**Pull request by bryanyzhu**
This PR is to introduce AutoGluon in GluonCV. The support for image classification and object detection is better with AutoGluon MultiModalPredictor.

### CODE REVIEW

File: gluoncv/torch/data/video_cls/multigrid_helper.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import numpy as np
 
 from torch.utils.data import Sampler
-from torch._six import int_classes as _int_classes
+# from torch._six import int_classes as _int_classes
```
Comment (yinweisu) [inline gluoncv/torch/data/video_cls/multigrid_helper.py:6]: Do we want to leave this as comment or just remove it?

**Review by yinweisu — APPROVED** [review]
LGTM! One minor comment

File: README.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ models. This toolkit offers four main features:
 4. Carefully designed APIs that greatly reduce the implementation complexity
 5. Community supports
 
+Please also checkout [AutoGluon](https://github.com/autogluon/autogluon) if you have image classification or object detection need. We have built the [MultimodalPredictor](https://auto.gluon.ai/stable/tutorials/multimodal/index.html) with improved model zoo, including TIMM, Huggingface, MMDetection and more. With just a few lines of code, you can train and deploy high accuracy computer vision models for your application.
```
Comment (Innixma) [inline README.md:29]: or object detection need -> or object detection needs

Comment (Innixma) [reply README.md:29]: image classification -> add tutorial link

Comment (Innixma) [reply README.md:29]: object detection -> add tutorial link

Comment (Innixma) [reply README.md:29]: with an improved model zoo

File: docs/tutorials/index.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@ Apache MXNet Tutorials
 
 Interested in getting started in a new CV area? Here are some tutorials to help get started.
 
+Note: For image classification or object detection task, feel free to explore tutorials in
```
Comment (Innixma) [inline docs/tutorials/index.rst:6]: or object detection task -> or object detection tasks

**Review by Innixma — COMMENTED** [review]
Added comments on doc changes

**Pull request merged**